src/aks-preview/azext_aks_preview/tests/latest/test_aks_commands.py

```diff
@@ -370,6 +370,27 @@ def test_aks_create_with_openservicemesh_addon(self, resource_group, resource_gr
             self.check('addonProfiles.openServiceMesh.enabled', True),
         ])
 
+    @AllowLargeResponse()
+    @AKSCustomResourceGroupPreparer(random_name_length=17, name_prefix='clitest', location='westus2')
+    def test_aks_addon_list(self, resource_group, resource_group_location):
```

Comment:
Please add more test cases to cover other new commands (enable, disable, update).